### DISCUSSION TITLE
Disable webproxy

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#3867] [Q&A]
Hello,
I use Azuracast on an Ubuntu 16.04 server in the Docker variant. 

I wanted to ask if it is possible to disable the webproxy completely, so that radios are no longer accessible under /radio/8000 or whatever. 

Greetings

**Comment by BusterNeece** [comment]
It is not possible currently to completely disable the web proxies; they're enabled regardless of whether they are actively in use by the station for stream playback URLs. You can disable showing those URLs across your application by disabling the `Use Web Proxy` setting in the system administration, though.
